chcemvediet/apps/inforequests/templates/inforequests/detail/action/main.html

```diff
@@ -14,8 +14,9 @@
 {% endcomment %}
 
 
-<div id="a{{ action.pk }}">
-  {% with print_id=idgenerator.next %}
+{% with action_id='a'|add:action.pk|stringformat:"s" %}
```

Comment:
`string|add:int` nefunguje ako by si cakal. Mozno v novsej verzii Djanga to uz funguje, ale v pouzivanej 1.7 to skonci tak, ze do `action_id` sa priradi prazdny string.

Reply:
Aha, OK, 1.7.